src/foam/u2/view/LoginView.js

```diff
@@ -217,7 +218,10 @@ foam.CLASS({
     {
       class: 'foam.u2.ViewSpec',
       name: 'leftView',
-      documentation: 'Allows using U2 views as left half of the login page, takes precedence over imgPath'
+      documentation: 'Allows using U2 views as left half of the login page, takes precedence over imgPath',
+      factory: function() {
+        return this.ctrl?.loginView?.leftView;
```

Comment:
if this a proper way?

Reply:
Why is this needed?

Reply:
when I clicked "sign in" button in top left corner from the unauthenticated-transfer, the view on the left is LoginView and the menuid is `sign-in`

Reply:
Right, okay, this is fine